deepfillv2/trainer.py

```diff
@@ -12,9 +12,12 @@
 import network
 import dataset
 import utils
+import wandb

```

Comment:
I think the better solution is
```suggestion
try:
    import wandb
except ModuleNotFoundError:
    wandb = None
```
this doesn't require user to have wandb installed if he doesn't want to.
Then, in other places, the following snippet may be used
```python
if wandb is not None:
    # code goes here
```

Reply:
Ah sorry, i committed too many things to master. i'll update this PR this week and get back to you!